llvm-bazel/llvm_configs/abi-breaking.h.cmake

```diff
@@ -20,7 +20,7 @@
 
 /* Allow selectively disabling link-time mismatch checking so that header-only
    ADT content from LLVM can be used without linking libSupport. */
-#if !LLVM_DISABLE_ABI_BREAKING_CHECKS_ENFORCING
+#if !defined(LLVM_DISABLE_ABI_BREAKING_CHECKS_ENFORCING) || !LLVM_DISABLE_ABI_BREAKING_CHECKS_ENFORCING
```

Comment:
Do we not need to update our `abi-breaking.h` file?

Reply:
We don't have one. We just have a Bazel rule that depends on the specific structure (but not this part of it)

https://github.com/google/llvm-bazel/blob/main/llvm-bazel/llvm-project-overlay/llvm/BUILD#L113-L124

Reply:
Doh, sorry, forgot this one worked differently. Should have checked.

(Also sorry I didn't see your response earlier.)

Reply:
No worries